src/core/mocks/domains/examine-management.handlers.ts

```diff
@@ -0,0 +1,76 @@
+import { rest } from 'msw';
+import { searchResultMockData, getIndexByName, PagedIndexers } from '../data/examine.data';
+
+import { umbracoPath } from '@umbraco-cms/utils';
+import { Index, PagedIndex, PagedSearcher, PagedSearchResult, SearchResult } from '@umbraco-cms/backend-api';
+
+export const handlers = [
+	rest.get(umbracoPath('/search/index'), (_req, res, ctx) => {
+		return res(
+			// Respond with a 200 status code
+			ctx.status(200),
+			ctx.json<PagedIndex>(PagedIndexers)
+		);
+	}),
+
+	// TODO: when using the umbracoPath helper you have to write parameters like {indexName}. MSW wants parameters as :indexName
+	rest.get('/umbraco/management/api/v1/search/index/:indexName', (_req, res, ctx) => {
```

Comment:
The `umbracoPath` function just prepends the base URL to the path. So it should work even with `:indexName`. Are you sure this does not work?

```suggestion
	rest.get(umbracoPath('/search/index/:indexName'), (_req, res, ctx) => {
```

Reply:
Seems to work, not sure why I didnt test it after we got the new API